addons/mail/static/tests/chatter/web/chatter_tests.js

```diff
@@ -909,3 +909,20 @@ QUnit.test("Mentions in composer should still work when using pager", async () =
     await insertText(".o-mail-Composer-input", "@");
     await contains(".o-mail-Composer-suggestion", { count: 2 });
 });
+
+QUnit.only(
+    "should display the subject even if the record name is false",
+    async () => {
+        const pyEnv = await startServer();
+        const fakeId = pyEnv["res.fake"].create({ name: false });
+        pyEnv["mail.message"].create({
+            body: "not empty",
+            model: "res.fake",
+            res_id: fakeId,
+            subject: "Salutations, voyageur",
+        });
+        const { openFormView } = await start();
+        openFormView("res.fake", fakeId);
+        await contains(".o-mail-Message", { text: "Subject: Salutations, voyageurnot empty" });
+    }
+);
```

Comment:
Not for `17.2` fw-port: tests have been converted to HOOT.
Thus add test in existing file `chatter.test.js`.

Test is mostly unchanged: `QUnit` => `test` & `openFormView` is global import so no need to retrieve from `start()`.